tests/ut_mdesktopentry.cpp

```diff
@@ -396,6 +395,7 @@ void UtMDesktopEntry::localization()
 
     if (!lang.isEmpty()) {
         qputenv("LANG", lang.toLocal8Bit());
+        qputenv("LC_ALL", lang.toLocal8Bit());
```

Comment:
And to quote SUSv2:

<q>     This variable determines the values for all locale categories. The value of the LC_ALL environment variable has precedence over any of the other environment variables starting with LC_ (LC_COLLATE, LC_CTYPE, LC_MESSAGES, LC_MONETARY, LC_NUMERIC, LC_TIME) and the LANG environment variable. </q>

(http://pubs.opengroup.org/onlinepubs/7908799/xbd/envvar.html)